README.rst

```diff
@@ -76,20 +81,35 @@ Creating an instance::
     >>> res1 = Ok('yay')
     >>> res2 = Err('nay')
 
-Or through the class methods::
-
-    >>> from result import Result
-    >>> res1 = Result.Ok('yay')
-    >>> res2 = Result.Err('nay')
-
-Checking whether a result is ``Ok`` or not::
+Checking whether a result is `Ok` or `Err`. With ``isinstance`` you get type safe
+access that can be checked with MyPy. The `is_ok()` or `is_err()` methods can be
+used if you don't need the type safety with MyPy::
 
     >>> res = Ok('yay')
+    >>> isinstance(res, Ok)
+    True
+    >>> isinstance(res, Err)
+    False
     >>> res.is_ok()
     True
     >>> res.is_err()
     False
 
+You can also check if an object is `Ok` or `Err` by using the `Result_` type.
+Please note that this type is designed purely for convenience, and should not be used
+for anything else. Using `(Ok, Err)` also works fine::
```

Comment:
Since the README is in ReStructuredText format, you need to use double backticks everywhere for inline code. There are probably some other cases as well that need to be fixed.

Reply:
Ah, so used to markdown. Fixed in 688711e

Reply:
Yep, this gets me too :slightly_smiling_face: Only reason to use RST is that it can be directly integrated into the package description in `setup.py`. (Although nowadays markdown might be supported as well...)